tests/lib/nested.sh

```diff
@@ -415,10 +415,23 @@ nested_secboot_remove_signature() {
 }
 
 nested_secboot_sign_file() {
-    local FILE="$1"
-    local KEY="$2"
-    local CERT="$3"
-    nested_secboot_remove_signature "$FILE"
+    args=()
+    while [ "${#}" -gt 0 ]; do
+        case "${1}" in
+            --keep-signatures)
+                keep_signatures=1
```

Comment:
may this need `local keep_signatures` ? probably doesn't matter much since it's all invoked through `tests.nested wrapper`